File: src/main/java/unq/edu/ar/grupoMs12021/ServletInitializer.java
package unq.edu.ar.grupoMs12021;

import org.springframework.boot.builder.SpringApplicationBuilder;
import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
import unq.edu.ar.grupoMs12021.GrupoMs12021App;

public class ServletInitializer extends SpringBootServletInitializer {

	@Override
	protected SpringApplicationBuilder configure(SpringApplicationBuilder application) {
		return application.sources(GrupoMs12021App.class);
	}

}

File: src/main/kotlin/unq/edu/ar/GrupoMs12021/Resenia/webservice/ServletInitializer.kt
package unq.edu.ar.GrupoMs12021.Resenia.webservice

import org.springframework.boot.builder.SpringApplicationBuilder
import org.springframework.boot.web.servlet.support.SpringBootServletInitializer

class ServletInitializer : SpringBootServletInitializer() {

	override fun configure(application: SpringApplicationBuilder): SpringApplicationBuilder {
		return application.sources(ReseniaApplication::class.java)
	}

}
